fdbserver/MoveKeys.actor.cpp

```diff
@@ -1542,13 +1556,15 @@ void seedShardServers(Arena& arena,
 		           : keyServersValue(RangeResult(), { serverSrcUID[serverIndex] }, serverTeamIDs[serverIndex]);
 	};
 
+	auto keySplits = keySplits_;
+	keySplits.push_back("\xff"_sr);
 	const int numServers = servers.size();
 	const int numKeyRanges = keySplits.size() + 1;
 
 	ASSERT(numKeyRanges > 1);
 	ASSERT(numKeyRanges <= numServers);
 
-	int serverIndex = 0;
+	int serverIndex = 1;
```

Comment:
Maybe add a comment, explaining the purpose is to assign `\xff` to the `txsTeam`.